soaks/bin/run_experiment.sh

```diff
@@ -125,6 +126,10 @@ do
     fi
     recorded_samples=$observed_samples
     (( periods = periods + 1 ))
+    if [ $periods -gt $SAMPLE_ATTEMPT_LIMIT ]; then
+        echo "INSUFFICIENT SAMPLES COlLECTED BEFORE DEADLINE. THIS INDICATES A PROBLEM WITH THE EXPERIMENT."
```

Comment:
As an enhancement, we could drop a `kubectl log` in here of the vector container which will probably provide useful error output.

Reply:
@spencergilbert has brought that up before. We need to amend the AMI to have kubectl installed is all.